pyface/ui/wx/action/menu_manager.py

```diff
@@ -103,14 +103,20 @@ def __init__(self, manager, parent, controller):
         self.menu_items = []
 
         # Create the menu structure.
-        self.refresh()
+        self.refresh(event=None)
```

Comment:
refresh signature has been updated so now we can remove this 
```suggestion
        self.refresh()
```